src/core/core.layoutService.js

```diff
@@ -172,6 +172,12 @@ module.exports = function(Chart) {
 			// Step 1
 			var chartWidth = width - leftPadding - rightPadding;
```

Comment:
A simpler way to do this more inline with existing conventions would be:

    var chartWidth = Math.min(width - leftPadding - rightPadding, 0);